client/src/components/pages/PageNotFound.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+class Landing extends Component {
```

Comment:
should be PageNotFound

Reply:
Fixed. I'm not sure why it still worked

Reply:
I think it's because when you import it, you import it as PageNotFound